custom/ilsgateway/migrations/0012_remove_like_indexes.py

```diff
@@ -0,0 +1,202 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        db.execute("DROP INDEX IF EXISTS ilsgateway_alert_supply_point_like")
+        # db.execute("DROP INDEX IF EXISTS ilsgateway_deliverygroupreport_message_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_deliverygroupreport_supply_point_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_organizationsummary_supply_point_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_productavailabilitydata_product_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_productavailabilitydata_supply_point_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_requisitionreport_location_id_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_supplypointstatus_supply_point_like")
+        db.execute("DROP INDEX IF EXISTS ilsgateway_supplypointwarehouserecord_supply_point_like")
```

Comment:
:+1: cc @kkrampa but I'm pretty sure we don't do any like queries anywhere in the ils/ews code